packages/manager/modules/emailpro/src/domain/dkim-autoconfig/emailpro-domain-dkim-autoconfig.constants.js

```diff
@@ -32,6 +32,43 @@ export const DKIM_CONFIGURATION_GUIDE = {
   DEFAULT: 'https://help.ovhcloud.com/csm#knownErrors',
 };
 
+export const DKIM_CONFIGURATION_GUIDE_NO_OVH = {
+  US:
```

Comment:
ask for the good US guide link. It should start with support.us.ovhcloud.com

Reply:
I ask, ty